config/params.ini

```diff
@@ -89,6 +89,11 @@ id_items=1
 open_chests=1
 fill_shared_stash_first=0
 
+[gamble]
+gamble=1, 
```

Comment:
let's make it opt-in to gamble, so default `=0`, what do you think?

Reply:
i agree. Default should ge gamble=0